src/node/run-style-dictionary.js

```diff
@@ -185,7 +185,7 @@ export default async function runStyleDictionary() {
     // If .js, we need to parse it as actual JS without resorting to eval/Function
     // Instead, we put it in a blob and create a URL from it that we can import
     // That way, malicious code would be scoped only to the blob, which is safer.
-    if (configPath.endsWith(".js")) {
+    if (configPath.match(/.(m)?js$/g)) {
```

Comment:
and the capturing group is not needed too

Reply:
Ah good catch. I had the group there because first I tried to see if CJS would work but I don't have rollup commonjs working in the browser here as opposed to backlight where it does work (I think via esbuild though)